core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-store/dot-pages.store.ts

```diff
@@ -815,7 +815,7 @@ export class DotPageStore extends ComponentStore<DotPagesState> {
         if (isEnterprise && environments) {
             actionsMenu.push({
                 label: this.dotMessageService.get('contenttypes.content.push_publish'),
-                command: (item) =>
+                command: () =>
```

Comment:
We should create a test for this to avoid a bug by mistake.

Reply:
Yes

Reply:
Done!